src/server/engine_shard_set.cc

```diff
@@ -573,8 +551,6 @@ void EngineShard::PollExecution(const char* context, Transaction* trans) {
     // Otherwise it is required to stay there to keep the relative order.
     if (is_ooo && !trans->IsMulti())
       DCHECK_EQ(keep, trans->GetLocalTxqPos(sid) != TxQueue::kEnd);
-
-    DLOG_IF(INFO, !dbg_id.empty()) << "Eager run " << sid << ", " << dbg_id << ", keep " << keep;
```

Comment:
Did you intentionally remove this? I don't see this moved anywhere.

Reply:
Its part of run() now 🙂 Instead of different names I just print the ooo flag